### PR TITLE
Management cli

### PR DESCRIPTION
**Pull request by asafshen**
## Related Issues

Fixes https://github.com/descope/etc/issues/2970

## Description
management sdk built on top of the popular [Commander CLI tool](https://github.com/tj/commander.js/)

instructions in readme

### CODE REVIEW

**Comment by dorsha** [comment]
Let's change to `Cli`

**Review by dorsha — COMMENTED** [review]
Cool, just make sure lint is running on that folder.

**Comment by asafshen** [comment]
> Let's change to Cli in code occurrences.

done (expect titles etc)
(fyi git does not catch CLI -> Cli so I had to rename to an intermediate -> commit -> rename again 😄 )



> Cool, just make sure lint is running on that folder.

it does not
image

but I will change in a separate PR, there are more examples and fixes so I prefer to push separately

File: examples/managementCli/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+  .description('Load for all tenants')
+  .action(async () => {
+    handleSdkRes(await sdk.management.tenant.loadAll());
+  });
+
+// *** Group commands ***
+
+// group-all-for-tenant
+program
+  .command('group-all-for-tenant')
```
Comment (orius123) [inline examples/managementCli/src/index.ts:278]: Missing from readme, can we generate the docs somehow? or have usage command

Comment (asafshen) [reply examples/managementCli/src/index.ts:278]: there is an help command which also appears in the readme
image

I don't want to auto-generate the readme from the command, its an overkill imo especially for management cli example

but I will add those commands to readme as well

**Review by orius123 — APPROVED** [review]

**Pull request merged**